### PR TITLE
Added missing this in geometry tests

### PR DESCRIPTION
**Pull request by davidpiuva**
Adding lots of missing this for geometry. Some overlapping names were changed for clarity.

### CODE REVIEW

**Comment by davidpiuva** [comment]
@sebastianbaginski Peer review?

**Comment by sebastianbaginski** [comment]
Looks good.

**Comment by davidpiuva** [comment]
Static calls were missed because the compiler resolved it as a namespace. More might come.

**Comment by davidpiuva** [comment]
Updated with missing This for the expect method.

**Comment by davidpiuva** [comment]
Promise failed. I try again.

**Comment by davidpiuva** [comment]
Updated tolerance.

File: test/geometry/FloatPoint2DTest.ooc [hunk 1]
```diff
@@ -86,15 +86,15 @@ FloatPoint2DTest: class extends Fixture {
 			expect(point azimuth, is equal to(Float pi))
 		})
 		this add("polar 5", func {
-			point := FloatPoint2D new(-3, 0)
-			point2 := FloatPoint2D polar(point norm, point azimuth)
-			expect(point distance(point2), is equal to(0.0f) within(this precision))
+			pointA := FloatPoint2D new(-3, 0)
+			pointB := FloatPoint2D polar(pointA norm, pointA azimuth)
```
Comment (marcusnaslund) [inline test/geometry/FloatPoint2DTest.ooc]: Good.

**Pull request merged**